dvc/tree/gdrive.py

```diff
@@ -510,35 +511,68 @@ def exists(self, path_info, use_dvcignore=True):
         else:
             return True
 
-    def _list_paths(self, prefix=None):
-        if not self._ids_cache["ids"]:
-            return
+    def ls(
+        self, path_info, detail=False, recursive=False
+    ):  # pylint: disable=arguments-differ
+        if recursive:
+            for entry in self.walk_files(path_info, detail=detail):
+                if detail:
+                    yield entry
+                else:
+                    yield str(entry)
+            return None
+
+        parent_ids = self._path_to_item_ids(path_info.path)
```

Comment:
minor: a bit confusing name here - they are just ids here, not parent ids as far as I understand